### ISSUE TITLE
Online demo down?

### ISSUE THREAD

**Issue by hasan-sayeed** [imoneoi/openchat#39]
I used the demo yesterday. But I can't use it now. Is it down?

**Comment by imoneoi** [comment]
Tested it out, it seems to work now

**Comment by hasan-sayeed** [comment]
Thank you for your reply. I don't know why it's not working for me. Is there any limit of usage or something like that?

**Comment by imoneoi** [comment]
No, there are no restrictions. You can try a different browser or Internet connection.
